tube/spark/plugins/post_process.py

```diff
@@ -0,0 +1,9 @@
+def add_auth_resource_path(df):
+    # PXD-1688
```

Comment:
can we get rid of jira ticket in code?
external people don't have access to jira. but this open source repository should still makes sense

Reply:
Yes, sure.